index.bs

```diff
@@ -3967,6 +3987,24 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
         interest of minimizing disruption to the user interaction flow).
 </div>
 
+### Client Capabilities Enumeration (enum <dfn enum>ClientCapabilities</dfn>) ### {#enum-clientCapabilities}
+
+<xmp class="idl">
+    enum ClientCapabilities {
+        "hybrid-transport",
+        "client-pin-entry"
+    };
+</xmp>
+
+This enumeration defines a limited set of client capabilities which a [=[WRP]=] may evaluate to offer certain workflows and experiences to users.
+
```

Comment:
Add a "Note:" about the enum not being referenced, as done for other enums? ([Example](https://www.w3.org/TR/webauthn-2/#enum-attachment))